espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/handlers/CreateSession.kt

```diff
@@ -41,7 +41,8 @@ class CreateSession : RequestHandler<SessionParams, Session> {
                 startActivity(
```

Comment:
I would rather change startActivity signature to accept a data object. four arguments are too many

Reply:
Yes. I also want to have a data object. But i don't know how to write.

Reply:
You could simply reuse StartActivityParams one